torch/nn/modules/dropout.py

```diff
@@ -181,6 +181,42 @@ def forward(self, input):
 
 
 class FeatureAlphaDropout(_DropoutNd):
+    r"""Randomly zero out entire channels (a channel is a 3D feature map,
```

Comment:
This doesn't describe the Alpha part. Also need ref to the SELU paper.

Reply:
Addressed in next update.